devtools/x/src/diff_summary.rs

```diff
@@ -15,9 +24,9 @@ pub struct Args {
     #[structopt(name = "COMPARE_SUMMARY")]
     /// Path to the comparison summary
     compare_summary: PathBuf,
-    #[structopt(name = "OUTPUT_FORMAT")]
+    #[structopt(name = "OUTPUT_FORMAT", default_value = "Text")]
```

Comment:
This should also have a `possible_values` argument: https://github.com/diem/diem/blob/39e4d7a064ea4b9d071c56d9df5099b655c4b8f8/devtools/x/src/cargo/build_args.rs#L85

Reply:
I don't think we need this for an enum:

```sh
> cargo x generate-summaries out_dir bad_value
error: Invalid value for '<OUTPUT_FORMAT>': valid values: Toml, Json

> cargo x generate-summaries -h
Generate build summaries for important subsets

USAGE:
    x generate-summaries [ARGS]

FLAGS:
    -h, --help       Prints help information
    -V, --version    Prints version information

ARGS:
    <OUT_DIR>          Directory to output summaries to (default: target/summaries)
    <OUTPUT_FORMAT>    Output in text, toml or json [default: Toml]
```

Reply:
Ahh perfect! Thanks.